### PR TITLE
Added Bucket_Sort C# #2124

### LINKED ISSUE #2124: Bucket Sort
- [x] C
- [x] C++
- [ ] C# - @luv4bytes
- [ ] CoffeeScript
- [ ] Dart
- [x] Go - @m-code12
- [x] Java
- [x] JS - @abhu-A-J
- [x] Kotlin - @shubhammantri1
- [ ] Php - @srimani-programmer
- [x] Python
- [x] Readme
- [x] Ruby - @m-code12
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by luv4bytes**
Fixes #2124 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Bucket_Sort in C#
#### Languages Used:
- C#

#### Files Added:
- Bucket_Sort.cs

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by luv4bytes** [comment]
Hello @jainaman224, please review.

File: Bucket_Sort/Bucket_Sort.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            }
+
+            return sorted;
+        }
+
+        /// Do BucketSort.
+        static void Main(string[] args)
+        {
+            int buckets = 5;
+            int[] input = new int[] {2, 16, 3, 82, 7, 132, 5200, 6, 4, 10000, 8, 9};
```
Comment (jainaman224) [inline Bucket_Sort/Bucket_Sort.cs]: Take input from user.

**Comment by raghav-dalmia** [comment]
Are you working on it?

**Comment by luv4bytes** [comment]
Sorry, haven't seen the message, yes I'll start applying the changes.

**Comment by luv4bytes** [comment]
@jainaman224 applied the change

**Comment by luv4bytes** [comment]
@jainaman224 just wanted to mention that the review ist still requested :)

File: Bucket_Sort/Bucket_Sort.cs [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        * Buckets:
+        * [0] = [2, 3, 4, 6, 7, 8, 9, 16, 82, 132]
+        * [1] = []
+        * [2] = [5200]
+        * [3] = []
+        * [4] = [10000]
+        *
+        * Output: [2, 3, 4, 6, 7, 8, 9, 16, 82, 132, 5200, 10000]
+        */
+
```
Comment (asha15) [inline Bucket_Sort/Bucket_Sort.cs:118]: take this I/O o the end of the file

**Pull request closed without merging**